### ISSUE TITLE
Album tags

### ISSUE THREAD

**Issue by rsinghal** [ProjectMirador/mirador-design#16]
In the Catalog mockups (19), there are tags for the album.  Where are those coming from?

**Comment by ggeisler** [comment]
We discussed this on the last call, but just to clarify more broadly, the mockups are admittedly confusing in showing tags, in that album/manifest tags are probably only applicable in a Mirador as a Service context, where the tags would be used by the user for local organization and stored in a database or something. 

To prevent future confusion, I'll remove tags from the catalog mockups. When we get to mockups that show specific Mirador as a Service features we can reintroduce the tags idea there.

To be clear, though, album tags are included the requirements we created as part of the discovery process (see lines 70, 85, and 86 in the [requirements spreadsheet](https://docs.google.com/spreadsheets/d/1W-ppSsRz0V2UX1nBkqIPKdIu5-4uJX3_B3BXEJH2WJM/) where we sometimes used the term "collections" to mean albums/manifests).

**Comment by azaroth42** [comment]
Tags can be implemented in IIIF using Annotations. Introducing a non-conforming way to implement something that can be done already would be bad for the community.

**Comment by ggeisler** [comment]
@azaroth42 Can you point to an example of where annotation tags are used to provide a descriptive tag for a manifest (e.g., tagging an object, not a sing annotation)?

Also, it's not clear what you are giving a thumbs-down to in my comment above. Are you suggesting the user requirements we gathered are not valid or something?

**Comment by azaroth42** [comment]
No, because there's no consuming implementation and hence no one has added tags in the wild.

I'm :-1: that tags are a "specific Mirador as a Service feature". They're perfectly possible today using 2.1, and even cleaner / more obvious in 3.0.
